modules/@apostrophecms/page/index.js

```diff
@@ -1625,14 +1641,27 @@ database.`);
             parent = target._ancestors[0];
           }
           async function nudgeNewPeers() {
-            // Nudge down the pages that should now follow us
-            await self.apos.doc.db.updateMany({
+            const criteria = {
```

Comment:
@boutell Isn't that a bug? The criteria doesn't involve any locale filter, which would eventually touch pages from a different locale?

Reply:
I think you are right, I think there is a bug here.

I think it should work like the correctly written query on the `const modified =` line.

Reply:
I only moved the locale criteria, which was the main issue. The moved document ID is mutated after nudging the peers and works correctly so I'll leave it be to not introduce regressions by an accident.